*
* Visual Studio Code Server
*
* By using the software, you agree to
* the Visual Studio Code Server License Terms (https://aka.ms/vscode-server-license) and
* the Microsoft Privacy Statement (https://privacy.microsoft.com/en-US/privacystatement).
*
Server bound to 127.0.0.1:38117 (IPv4)
Extension host agent listening on 38117

[02:12:20] Extension host agent started.
[02:12:20] [127.0.0.1][9cac0b0f][ManagementConnection] New connection established.
[02:12:20] [127.0.0.1][8841f935][ExtensionHostConnection] New connection established.
[02:12:20] [127.0.0.1][8841f935][ExtensionHostConnection] <148472> Launched Extension Host Process.
[02:27:06] [127.0.0.1][8841f935][ExtensionHostConnection] The client has reconnected.
[02:27:10] [127.0.0.1][9cac0b0f][ManagementConnection] The client has disconnected, will wait for reconnection 3h before disposing...
[02:27:10] Error: Unexpected SIGPIPE
    at process.<anonymous> (/home/fred/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/server/node/server.main.js:143:7923)
    at process.emit (node:events:390:28)
[05:03:06] [127.0.0.1][9cac0b0f][ManagementConnection] Another client has connected, will shorten the wait for reconnection 5m before disposing...
[05:03:06] [127.0.0.1][14591159][ManagementConnection] New connection established.
[05:03:06] [127.0.0.1][4cb9340b][ExtensionHostConnection] New connection established.
[05:03:06] [127.0.0.1][4cb9340b][ExtensionHostConnection] <149764> Launched Extension Host Process.
[IPC Library: Pty Host]  INFO Persistent process "6": Replaying 3808 chars and 1 size events
rejected promise not handled within 1 second: Error: Could not find pty on pty host
stack trace: Error: Could not find pty on pty host,    at w._throwIfNoPty (/home/fred/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:20:5333),    at w.updateTitle (/home/fred/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:20:1673),    at Object.call (/home/fred/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:13:8386),    at h.onPromise (/home/fred/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:12:4891),    at h.onRawMessage (/home/fred/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:12:4274),    at /home/fred/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:12:3560,    at E.invoke (/home/fred/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:8:145),    at W.deliver (/home/fred/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:8:2271),    at c.fire (/home/fred/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:8:1849),    at process.U (/home/fred/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:6:35698),    at process.emit (node:events:390:28),    at emit (node:internal/child_process:917:12),    at processTicksAndRejections (node:internal/process/task_queues:84:21)
[05:03:17] [Error: Could not find pty on pty host,    at w._throwIfNoPty (/home/fred/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:20:5333),    at w.updateTitle (/home/fred/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:20:1673),    at Object.call (/home/fred/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:13:8386),    at h.onPromise (/home/fred/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:12:4891),    at h.onRawMessage (/home/fred/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:12:4274),    at /home/fred/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:12:3560,    at E.invoke (/home/fred/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:8:145),    at W.deliver (/home/fred/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:8:2271),    at c.fire (/home/fred/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:8:1849),    at process.U (/home/fred/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:6:35698),    at process.emit (node:events:390:28),    at emit (node:internal/child_process:917:12),    at processTicksAndRejections (node:internal/process/task_queues:84:21)]
rejected promise not handled within 1 second: Error: Could not find pty on pty host
stack trace: Error: Could not find pty on pty host,    at w._throwIfNoPty (/home/fred/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:20:5333),    at w.updateIcon (/home/fred/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:20:1733),    at Object.call (/home/fred/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:13:8386),    at h.onPromise (/home/fred/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:12:4891),    at h.onRawMessage (/home/fred/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:12:4274),    at /home/fred/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:12:3560,    at E.invoke (/home/fred/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:8:145),    at W.deliver (/home/fred/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:8:2271),    at c.fire (/home/fred/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:8:1849),    at process.U (/home/fred/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:6:35698),    at process.emit (node:events:390:28),    at emit (node:internal/child_process:917:12),    at processTicksAndRejections (node:internal/process/task_queues:84:21)
[05:03:17] [Error: Could not find pty on pty host,    at w._throwIfNoPty (/home/fred/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:20:5333),    at w.updateIcon (/home/fred/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:20:1733),    at Object.call (/home/fred/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:13:8386),    at h.onPromise (/home/fred/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:12:4891),    at h.onRawMessage (/home/fred/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:12:4274),    at /home/fred/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:12:3560,    at E.invoke (/home/fred/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:8:145),    at W.deliver (/home/fred/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:8:2271),    at c.fire (/home/fred/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:8:1849),    at process.U (/home/fred/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:6:35698),    at process.emit (node:events:390:28),    at emit (node:internal/child_process:917:12),    at processTicksAndRejections (node:internal/process/task_queues:84:21)]
[05:08:06] [127.0.0.1][9cac0b0f][ManagementConnection] The reconnection short grace time of 5m has expired, so the connection will be disposed.
[05:08:06] [127.0.0.1][8841f935][ExtensionHostConnection] <148472> Extension Host Process exited with code: 0, signal: null.
[05:28:33] [File Watcher (parcel)] Unexpected error: std::bad_alloc (EUNKNOWN) (path: /home/fred)
[05:28:33] [File Watcher (universal)] restarting watcher after error: std::bad_alloc
[08:08:08] [127.0.0.1][14591159][ManagementConnection] The client has disconnected, will wait for reconnection 3h before disposing...
[08:08:09] Error: Unexpected SIGPIPE
    at process.<anonymous> (/home/fred/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/server/node/server.main.js:143:7923)
    at process.emit (node:events:390:28)
[08:10:40] [127.0.0.1][14591159][ManagementConnection] Another client has connected, will shorten the wait for reconnection 5m before disposing...
[08:10:40] [127.0.0.1][2ce6826d][ManagementConnection] New connection established.
[08:10:40] [127.0.0.1][9a577943][ExtensionHostConnection] New connection established.
[08:10:40] [127.0.0.1][9a577943][ExtensionHostConnection] <196023> Launched Extension Host Process.
[IPC Library: Pty Host]  INFO Persistent process "6": Replaying 6613 chars and 1 size events
rejected promise not handled within 1 second: Error: Could not find pty on pty host
stack trace: Error: Could not find pty on pty host,    at w._throwIfNoPty (/home/fred/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:20:5333),    at w.updateTitle (/home/fred/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:20:1673),    at Object.call (/home/fred/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:13:8386),    at h.onPromise (/home/fred/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:12:4891),    at h.onRawMessage (/home/fred/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:12:4274),    at /home/fred/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:12:3560,    at E.invoke (/home/fred/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:8:145),    at W.deliver (/home/fred/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:8:2271),    at c.fire (/home/fred/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:8:1849),    at process.U (/home/fred/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:6:35698),    at process.emit (node:events:390:28),    at emit (node:internal/child_process:917:12),    at processTicksAndRejections (node:internal/process/task_queues:84:21)
[08:10:46] [Error: Could not find pty on pty host,    at w._throwIfNoPty (/home/fred/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:20:5333),    at w.updateTitle (/home/fred/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:20:1673),    at Object.call (/home/fred/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:13:8386),    at h.onPromise (/home/fred/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:12:4891),    at h.onRawMessage (/home/fred/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:12:4274),    at /home/fred/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:12:3560,    at E.invoke (/home/fred/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:8:145),    at W.deliver (/home/fred/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:8:2271),    at c.fire (/home/fred/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:8:1849),    at process.U (/home/fred/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:6:35698),    at process.emit (node:events:390:28),    at emit (node:internal/child_process:917:12),    at processTicksAndRejections (node:internal/process/task_queues:84:21)]
rejected promise not handled within 1 second: Error: Could not find pty on pty host
stack trace: Error: Could not find pty on pty host,    at w._throwIfNoPty (/home/fred/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:20:5333),    at w.updateIcon (/home/fred/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:20:1733),    at Object.call (/home/fred/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:13:8386),    at h.onPromise (/home/fred/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:12:4891),    at h.onRawMessage (/home/fred/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:12:4274),    at /home/fred/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:12:3560,    at E.invoke (/home/fred/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:8:145),    at W.deliver (/home/fred/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:8:2271),    at c.fire (/home/fred/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:8:1849),    at process.U (/home/fred/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:6:35698),    at process.emit (node:events:390:28),    at emit (node:internal/child_process:917:12),    at processTicksAndRejections (node:internal/process/task_queues:84:21)
[08:10:46] [Error: Could not find pty on pty host,    at w._throwIfNoPty (/home/fred/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:20:5333),    at w.updateIcon (/home/fred/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:20:1733),    at Object.call (/home/fred/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:13:8386),    at h.onPromise (/home/fred/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:12:4891),    at h.onRawMessage (/home/fred/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:12:4274),    at /home/fred/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:12:3560,    at E.invoke (/home/fred/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:8:145),    at W.deliver (/home/fred/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:8:2271),    at c.fire (/home/fred/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:8:1849),    at process.U (/home/fred/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:6:35698),    at process.emit (node:events:390:28),    at emit (node:internal/child_process:917:12),    at processTicksAndRejections (node:internal/process/task_queues:84:21)]
[08:15:40] [127.0.0.1][14591159][ManagementConnection] The reconnection short grace time of 5m has expired, so the connection will be disposed.
[08:15:40] [127.0.0.1][4cb9340b][ExtensionHostConnection] <149764> Extension Host Process exited with code: 0, signal: null.
